api_functional_v2_test.go

```diff
@@ -181,6 +181,109 @@ func TestRemovePartiallyUploadedV2(t *testing.T) {
 	}
 }
 
+// Tests resumable put object cloud to cloud.
```

Comment:
Even test code requires comment. Description of what each test is trying to achieve.